solutions/src/test/java/ru/sberbank/school/task08/JacksonSaveGameManagerTest.java

```diff
@@ -0,0 +1,29 @@
+package ru.sberbank.school.task08;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class JacksonSaveGameManagerTest {
+
+    private final JacksonSaveGameManager manager = new JacksonSaveGameManager("f:/temp");
+
+    @Test
+    void saveGameSavesGame() throws SaveGameException {
+        manager.initialize();
+        List<GameObject> gameObjects = new ArrayList<>();
+        gameObjects.add(manager.createEntity(InstantiatableEntity.Type.BUILDING, InstantiatableEntity.Status.SPAWNED, 1));
+        gameObjects.add(manager.createEntity(InstantiatableEntity.Type.ENEMY, InstantiatableEntity.Status.DESPAWNED, 2));
```

Comment:
Line is longer than 120 characters (found 121).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1420478281>*